test/utils.jl

```diff
@@ -25,37 +28,50 @@ end
     @test _ones(Float32, 2) == (1.0f0, 1.0f0)
 end
 
-@testitem "Differentiation (EnzymeExt loaded)" setup=[Utils] begin
-    # supports_autoenzyme(::Type{<:Any})
-    @test MeshIntegrals.supports_autoenzyme(Nothing) == false
-
-    # _default_diff_method -- using type or instance, Enzyme-supported combination
+@testitem "Differentiation (MeshIntegralsEnzymeExt loaded)" setup=[Utils] begin
+    # supports_autoenzyme & _default_diff_method
+    # Nominal usage: Enzyme-supported geometry and FP type
     let sphere = Sphere(Point(0, 0, 0), 1.0)
         @test _default_diff_method(Meshes.Sphere, Float64) isa AutoEnzyme
         @test _default_diff_method(sphere, Float64) isa AutoEnzyme
     end
-
-    # _default_diff_method -- Enzyme-unsupported FP types
+    # supports_autoenzyme(::Type{<:Any}) for CI completeness
+    @test MeshIntegrals.supports_autoenzyme(Nothing) == false
+    # BezierCurve: blocked by upstream Meshes-Enzyme incompatibility
+    let curve = BezierCurve([Point(t, 0) for t in range(-π, π, length = 361)])
+        @test_throws "proven readonly" jacobian(curve, (0.5,), AutoEnzyme())
+        @test _default_diff_method(Meshes.BezierCurve, Float64) isa FiniteDifference
+    end
+    # Cylinder: blocked by version-dependent upstream Meshes-Enzyme incompatibility
+    let cyl = Cylinder(Point(0, 0, 0), Point(0, 0, 1), 2.0)
+        @test _default_diff_method(Meshes.Cylinder, Float64) isa FiniteDifference
+    end
+    # CylinderSurface: blocked by version-dependent upstream Meshes-Enzyme incompatibility
+    let cylsurf = CylinderSurface(Point(0, 0, 0), Point(0, 0, 1), 2.0)
+        @test _default_diff_method(Meshes.CylinderSurface, Float64) isa FiniteDifference
+    end
+    # ParametrizedCurve: parametric functions are user-defined, can't guarantee behavior
+    @test _default_diff_method(Meshes.ParametrizedCurve, Float64) isa FiniteDifference
+    # Non-Float64: other FP types not supported by Enzyme
     @test _default_diff_method(Meshes.Sphere, Float16) isa FiniteDifference
     @test _default_diff_method(Meshes.Sphere, BigFloat) isa FiniteDifference
 
-    # _default_diff_method -- geometries that currently error with AutoEnzyme
-    @test _default_diff_method(Meshes.BezierCurve, Float64) isa FiniteDifference
-    @test _default_diff_method(Meshes.CylinderSurface, Float64) isa FiniteDifference
-    @test _default_diff_method(Meshes.Cylinder, Float64) isa FiniteDifference
-    @test _default_diff_method(Meshes.ParametrizedCurve, Float64) isa FiniteDifference
-
     # FiniteDifference
-    @test FiniteDifference().ε ≈ 1e-6
+    let diff = FiniteDifference()
+        @test typeof(diff.ε) == Float64
+        @test diff.ε  ≈ 1e-6
```

Comment:
```suggestion
        @test diff.ε ≈ 1e-6
```
This is what the formatter suggests.